asdf_astropy/tags/tests/helpers.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+import numpy as np
+
+
+def run_schema_example_test(organization, standard, name, version, check_func=None):
```

Comment:
Does this do the same thing as the pytest asdf schema tester plugin?  It also runs examples.  Is this doing anything different?